hypervisor/src/arch/riscv64/aia.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2024 Institute of Software, CAS. All rights reserved.
```

Comment:
I think this commit message could be tweaked (or split into 2 - one adding the AIA and the other just adding the module).

Reply:
Noted 🫡